packages/nimble-components/src/rich-text-editor-viewer/specs/README.md

```diff
@@ -0,0 +1,335 @@
+
+### Non-goals
+
+-   The visual and interaction design for `nimble-rich-text-editor` and `nimble-rich-text-viewer` is out of scope for this spec component.
+-   Blazor integration will be out of scope for this component as we have less or no experience in Blazor component development or other
+    related technology. Blazor example app addition will also be not covered in this spec.
+-   Adding an example Angular client application for this component may not be included in the initial releases. However, the integration
+    of the rich-text-editor with Angular applications, achieved through a directive, will be developed following the completion of this
+    component.
+
+### Features
+
+The `nimble-rich-text-editor` supports the basic text formatting options that includes:
+
+1. Bold
+2. Italics
+3. Strikeout
+4. Numbered List
+5. Bulleted List
+
+This component will also offer APIs and interactive methods to format texts in the following ways:
+
+1. Using the buttons located below the editor to modify the text formatting for the selected texts.
+2. Using the keyboard shortcuts to modify the text formatting in the editor.
+3. Support to get markdown output from the editor.
+4. Support to convert the markdown to HTML to render it back in the `nimble-rich-text-viewer` component.
+
+#### _Additional features out of scope of this spec_
+
+-   Allowing the user to tag or mention by entering `@` in the editor and selecting the user name from the drop-down list.
+-   Support for adding images to the editor either by uploading or by pasting it.
+-   Support for adding hyperlinks to the existing words in the editor. However, adding raw links by pasting to the editor is supported.
+-   Support for underlining the text.
+-   Provide APIs to make the buttons in the toolbar located below the editor to be hidden or disable.
+
+### Risks and Challenges
+
+-   Visual design and interaction design for both the components are out of scope for the initial release. We will start with the existing
+    [mockup screens](https://www.figma.com/file/Q5SU1OwrnD08keon3zObRX/SystemLink?type=design&node-id=6280-94045) which is designed for
+    comments feature. At this moment, we cannot afford to cover all the corner case of visual design specs but we managed to refer from
+    other visual designs and try to incorporate with the nimble standards.
+-   Due to the immediate requirement for GM (`Comments Feature`), we are not adding any additional enhancements or feature requirements
+    apart from the mentioned features in the initial release.
+-   The mobile view of the component has not been designed yet, and we are actively collaborating with the design team to create basic mockup screens. We will
+    update this spec accordingly as progress is made.
+
+### Prior Art/Examples
+
+Sample Mockup Screenshots
+
+_`nimble-rich-text-editor`_
+
+![Editor](./spec-images/editor-sample.png)
+
+_`nimble-rich-text-viewer`_
+
+![Viewer](./spec-images/viewer-sample.png)
```

Comment:
Let's crop the top header division with username, date and edit/delete menu as per latest discussion of scoping it into application workflow.